package.json

```
@@ -35,6 +35,7 @@
     "watch": "lerna run watch --scope={'@qubic-connect/redirect,@qubic-connect/detect-iab,@qubic-connect/core'}",
     "release": "npm run build && lerna publish --force-publish",
     "prepare": "husky install",
+    "build:iab:test": "yarn build && cd examples/example-pure-js && yarn build:dev && cd build && { echo; echo; } | surge --domain qubic-connect-block.surge.sh && cd ../../..",
```

```suggestion
    "deploy:preview":  "yarn workspace @qubic-connect/example-pure-js build:dev && surge --domain qubic-connect-block.surge.sh "
```
1. 這次是 iab ，但是這個指令看起來比較像都要預覽，所以我覺得不用加 iab
2. 有 workspace 可以用
3. 為什麼你用 pipeline `|` 不是 `&&`
4. 是不是只有你有權限可以 deploy 這個 domain?
5. 這個指令我沒試，你跑跑看可不可以正常動